src/settings/storeconf.cpp

```diff
@@ -13,8 +15,8 @@ StoreConf::StoreConf(QWidget * parent)
     : QWidget (parent)
 {
     conFile = getPathToConFile();
-    if (!conFile.size()) // keep all fields with default values, create conf.json with these values
```

Comment:
To my opinion it is not user-friendly approach to prompt path to config file when app is launched.
I would suggest to provide user with an opportunity to set settings during run time

Reply:
removed this option